install-racket.sh

```diff
@@ -12,10 +12,11 @@ fi
 DL_BASE="https://mirror.racket-lang.org/installers"
 
 if [[ "$RACKET_VERSION" = "HEAD" ]]; then
+    NWU_BASE="http://plt.eecs.northwestern.edu/snapshots/current/installers"
```

Comment:
I'm asking because the title of the PR talked about fixing the URL to use https. So I don't know if leaving `http:` here was a mistake -- or if it was intentional and your idea is that people would opt into `https:` using an env var.

Reply:
Heh... oops, yes I did. I'll fix.